core-bundle/tests/Image/Studio/FigureTest.php

```diff
@@ -339,6 +343,10 @@ public function testGetLegacyTemplateData(array $preconditions, array $buildAttr
             $this->getFixturesDir()
         );
 
+        if (null !== $marginProperty) {
+            $this->expectDeprecation('%sUsing Contao\Controller::generateMargin is deprecated%s');
```

Comment:
```suggestion
            $this->expectDeprecation('Since contao/core-bundle 4.13: Using Contao\Controller::generateMargin is deprecated%s');
```
```suggestion
            $this->expectDeprecation('%sUsing Contao\Controller::generateMargin is deprecated%s');
```

Reply:
7398b9f1d